Tests/Data/TH2M/th2m.prj

```diff
@@ -0,0 +1,339 @@
+<?xml version="1.0" encoding="ISO-8859-1"?>
+<OpenGeoSysProject>
+    <mesh>line_60_heat.vtu</mesh>
+    <geometry>line_60_heat.gml</geometry>
+    <media>
+      <medium>  <!-- First (and in this example only) medium, i.e. the porous medium -->
+        <phases>
+          <phase>
+          <!-- First solid phase. There are (there have to be?) as many solid phases
+             as material groups!-->
+            <name>solid</name> <!-- phase name is optional at the moment... -->
+            <components>
+              <!-- different solids within the same elements are realized via
+                   components. -->
+              <component>
+                <!-- Components can have names, too. If no name is given, a
+                   custom (unspecified) component is created. All required
+                   properties have to be specified in the following subtree. -->
+                <properties>
+                  <property>
+                    <!-- Properties must have names and types. Name in this
+                       context implies the sort of the property, such as
+                       density, viscosity, and so on... -->
+                    <name>thermal_conductivity</name>
```

Comment:
It could be better if  the name, e.g.  `thermal_conductivity`,  is used as the tag name of  a `property` directly, like
```
<properties>
     <thermal_conductivity>
    ....
     </thermal_conductivity>
     ...
    <density>
   ....
    </density>
</properties>
```
which gives a clear structure for reading and checking.